src/njs_cnst.c

```diff
@@ -0,0 +1,621 @@
+        njs_string_kw("symbol", 0, 0),
+        njs_string_kw("swap16", 0, 0),
+        njs_string_kw("swap32", 0, 0),
+        njs_string_kw("swap64", 0, 0),
+        njs_string_kw("tan", 0, 0),
+        njs_string_kw("tanh", 0, 0),
+        njs_string_kw("test", 0, 0),
+        njs_string_kw("then", 0, 0),
+        njs_string_kw("times", 0, 0),
+        njs_string_kw("toDateString", 0, 0),
+        njs_string_kw("toExponential", 0, 0),
+        njs_string_kw("toFixed", 0, 0),
+        njs_string_kw("toISOString", 0, 0),
+        njs_string_kw("toJSON", 0, 0),
+        njs_string_kw("toLocaleDateString", 0, 0),
+        njs_string_kw("toLocaleString", 0, 0),
+        njs_string_kw("toLocaleTimeString", 0, 0),
+        njs_string_kw("toLowerCase", 0, 0),
+        njs_string_kw("toPrimitive", 0, 0),
+        njs_string_kw("toPrecision", 0, 0),
+        njs_string_kw("toReversed", 0, 0),
+        njs_string_kw("toSorted", 0, 0),
+        njs_string_kw("toSpliced", 0, 0),
+        njs_string_kw("toStringTag", 0, 0),
+        njs_string_kw("toString", 0, 0),
+        njs_string_kw("toTimeString", 0, 0),
+        njs_string_kw("toUpperCase", 0, 0),
+        njs_string_kw("toUTCString", 0, 0),
+        njs_string_kw("trimEnd", 0, 0),
+        njs_string_kw("trimStart", 0, 0),
+        njs_string_kw("trim", 0, 0),
+        njs_string_kw("trunc", 0, 0),
+        njs_string_kw("type", 0, 0),
+        njs_string_kw("usec", 0, 0),
+        njs_string_kw("unscopables", 0, 0),
+        njs_string_kw("unshift", 0, 0),
+        njs_string_kw("utf-8", 0, 0),
+        njs_string_kw("value", 0, 0),
+        njs_string_kw("valueOf", 0, 0),
+        njs_string_kw("values", 0, 0),
+        njs_string_kw("version", 0, 0),
+        njs_string_kw("version_number", 0, 0),
+        njs_string_kw("write", 0, 0),
+        njs_string_kw("writable", 0, 0),
+        njs_string_kw("writeDoubleBE", 0, 0),
+        njs_string_kw("writeDoubleLE", 0, 0),
+        njs_string_kw("writeFloatBE", 0, 0),
+        njs_string_kw("writeFloatLE", 0, 0),
+        njs_string_kw("writeIntBE", 0, 0),
+        njs_string_kw("writeIntLE", 0, 0),
+        njs_string_kw("writeInt16BE", 0, 0),
+        njs_string_kw("writeInt16LE", 0, 0),
+        njs_string_kw("writeInt32BE", 0, 0),
+        njs_string_kw("writeInt32LE", 0, 0),
+        njs_string_kw("writeInt8", 0, 0),
+        njs_string_kw("writeUIntBE", 0, 0),
+        njs_string_kw("writeUIntLE", 0, 0),
+        njs_string_kw("writeUInt16BE", 0, 0),
+        njs_string_kw("writeUInt16LE", 0, 0),
+        njs_string_kw("writeUInt32BE", 0, 0),
+        njs_string_kw("writeUInt32LE", 0, 0),
+        njs_string_kw("writeUInt8", 0, 0),
+        njs_string_kw("written", 0, 0),
+    }
+};
+
+
+njs_cnst_hash_chunk_t njs_cnst_hash_predefined = {
+    .descr = {
+        .hash_mask = NJS_CNST_N_HASH_MASK,
+        .elts_size = NJS_CNST_N_VALUES,
+        .elts_count = 0,
+        .elts_deleted_count = 0,
+    }
+};
+
+
+njs_lvlhsh_t njs_cnst_hash = {
```

Comment:
`njs_lvlhsh_t` is deprecated, for new code I propose to use only `njs_flathsh_t` typedef and APIs.
Right now we have a strage mix of `njs_lvlhsh_t` and `njs_flathsh_t` declarations.